lib/command.ts

```diff
@@ -1,4 +1,4 @@
-import { Dictionary, assertNotUndefined, assertNotTrue } from './common-types'
+import { Dictionary, assertNotStrictEqual } from './common-types'
```

Comment:
As I understood it, the main idea behind `import type` is to be able to import type only without importing real js stuff, nor executing any module initialization code (as the import disappears from the resulting js code).

As long as we do not have such an issue, would this really be a clarification?

Also note that:
- regular imports only importing interfaces (as are most of our type-only imports, until we move to classes) are already removed from the resulting js code
- the specific `./common-types` import you point at cannot be a type import, as it also imports a helper function

I think this may however make sense in a future work when we will work on replacing `self` stuff by real classes (to import only class types, and not the classes themselves).